### ISSUE TITLE
Error trying to calm pawn in certain break states

### ISSUE THREAD

**Issue by TheMysteriousX** [Weilbyte/SnapOut#15]
```Could not reserve Thing_MealSimple981340(current stack count: 27) (layer: null) for Kelly for job TakeInventory (Job_7605074) A=Thing_MealSimple981340 (now doing job TakeInventory (Job_7605074) A=Thing_MealSimple981340(curToil=-1)) for maxPawns 1 and stackCount -1. Existing reserver: Alligator doing job Ingest (Job_7605068) A=Thing_MealSimple981340 (toilIndex=4)
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

TryMakePreToilReservations() returned false for a non-queued job right after StartJob(). This should have been checked before. curJob=TakeInventory (Job_7605074) A=Thing_MealSimple981340
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)
```

Whole output is https://gist.github.com/HugsLibRecordKeeper/f42f45557ec41a8c9b9c3c91710564bc

Steam build, 1.1.2610 rev1053.

At the point that was happening, 'Alligator' was on a food binge mental break, and 'Kelly' was attempting to calm them. 'Alligator' wasn't binging on food right at that moment though, he was watching TV, so consume simple meal was the second item on his 'to do' list.

I've noticed the same behaviour with other intermittent breaks, like drug binges and corpse fascination.

In terms of mods that mess with queueing/breaks, I'm using 'Dub's Break Mod'.

Mods that mess with queuing are 'Pick up and haul', 'Jobs of Opportunity', 'Share the Load', 'Common Sense', 'Brr and Phew'.

**Comment by Weilbyte** [comment]
Thanks for reporting!    
Working on a surefire way to clear the queue.

**Comment by Weilbyte** [comment]
Should be fixed by 9ea761a. Pushed update to Steam and GitHub.
